### PR TITLE
[15.0][MIG] purchase_force_invoiced: Migration to 15.0

### PR DESCRIPTION
**Pull request by Olageibol**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please include #1531 ?

**Comment by sergio-teruel** [comment]
What about this PR?? @Olageibol

File: purchase_force_invoiced/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline purchase_force_invoiced/__manifest__.py:1]:
```suggestion
# Copyright 2019 ForgeFlow S.L.
```

File: purchase_force_invoiced/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* Jordi Ballester <jordi.ballester@eficent.com>
```
Comment (MiquelRForgeFlow) [inline purchase_force_invoiced/readme/CONTRIBUTORS.rst:1]:
```suggestion
* Jordi Ballester <jordi.ballester@forgeflow.com>
```

**Comment by victoralmau** [comment]
Already merged at https://github.com/OCA/purchase-workflow/pull/1516

**Pull request closed without merging**